src/storage/btree.h

```diff
@@ -242,7 +245,9 @@ struct btree_scan
   bool is_btid_int_valid;
   bool is_scan_started;
   bool force_restart_from_root;
-
+#if 1				/* defined(SUPPORT_DEDUPLICATE_KEY_MODE) */
```

Comment:
```suggestion
#if defined(SUPPORT_DEDUPLICATE_KEY_MODE)
```

Reply:
There are two macros declared in the btree.h file: BTREE_RESET_SCAN and BTREE_INIT_SCAN.
There is a part that is initialized inside this so that the variable is always declared.

This will be adjusted in a future code cleanup for SUPPORT_DEDUPLICATE_KEY_MODE.